summertime/model/single_doc/multilingual/mbart_model.py

```diff
@@ -0,0 +1,121 @@
+from transformers import MBartForConditionalGeneration, MBart50Tokenizer
+from .base_multilingual_model import MultilingualSummModel
+
+
+class MBartModel(MultilingualSummModel):
+    # static variables
+    model_name = "mBART"
+    is_extractive = False
+    is_neural = True
+    is_multilingual = True
+
+    lang_tag_dict = {
```

Comment:
Is this only applicable to mBART, if it is then it's okay. Otherwise maybe put this in the `MultilingualSummModel`?

Reply:
This is mBART specific, unfortunately.